catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -73,10 +69,6 @@ private static boolean isAnonymousReportEnabled() {
 		return preferences != null && preferences.getBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, false);
```

Comment:
are both isAnonymousReportEnabled and isReportingEnabled necessary? can we please combine this into one function when already touching this class.